src/holidays/fr.yaml

```diff
@@ -1013,30 +1242,59 @@ Mayotte:
       '2021': [2, 28, 3, 14]
       '2022': [2, 26, 3, 13]
       '2023': [2, 18, 3, 5]
+      '2024': []
+      '2025': [3, 1, 3, 9]
+      '2026': [2, 28, 3, 8]
     - name: Vacances de Pâques
       '2018': [5, 6, 5, 13]
       '2019': [4, 27, 5, 5]
       '2020': [5, 3, 5, 10]
       '2021': [5, 1, 5, 16]
       '2022': [4, 30, 5, 15]
       '2023': [4, 22, 5, 8]
-    - name: Pont de l'Ascension
+      '2024': []
+      '2025': [4, 26, 5, 11]
+      '2026': [4, 30, 5, 17]
+    - name: Pont de l'Ascension 
+      '2018': [5, 10, 5, 10]
+      '2019': [5, 30, 6, 2]
       '2020': [5, 21, 5, 24]
+      '2021': [5, 14, 5, 16]
+      '2022': [5, 26, 5, 27]
+      '2023': [5, 17, 5, 21]
+      '2024': [5, 8, 5, 12] # double-check
```

Comment:
If I interpret the following source correctly, the array should be empty:
https://vacances-scolaires-education.fr/ascension-2024.html